src/gmpv_preferences_dialog.c

```diff
@@ -173,6 +173,11 @@ static GtkWidget *build_page(	const PreferencesDialogItem *items,
 			gtk_widget_set_size_request(widget, 100, -1);
 			gtk_file_chooser_set_filename(chooser, filename);
 
+			void free_data(gpointer data, GClosure *closure)
```

Comment:
```suggestion
			static void free_data(gpointer data, GClosure *closure)
```
The function should be `static` since we're not using it anywhere else.